css/styles.css

```diff
@@ -46,4 +48,60 @@ a {
   margin-top: 1rem;
   margin-bottom: 1rem;
   letter-spacing: 0.35rem;
+  cursor: pointer;
+}
+
+.habit-form {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.row.habit {
+  display: flex;
+  flex: 1;
+  overflow: auto;
+  padding-top: 3rem;
+}
+
+#enter-habit {
```

Comment:
It's generally not a good idea to target elements by id on stylesheets. Use classes instead.